plugins/tiddlywiki/filesystem/filesystemadaptor.js

```diff
@@ -82,20 +82,20 @@ FileSystemAdaptor.prototype.saveTiddler = function(tiddler,callback,options) {
 		if(err) {
 			return callback(err);
 		}
-		$tw.utils.saveTiddlerToFile(tiddler,fileInfo,function(err,fileInfo) {
+		$tw.utils.saveTiddlerToFile(tiddler,fileInfo, (err,fileInfo) => {
```

Comment:
I'm afraid we can't use arrow functions in the core. We stick to ES5.1 (roughly) because it's important to retain compatibility with older browsers/JS runtimes.

Reply:
I will keep that in mind in the following PRs.